packages/@aws-cdk/app-staging-synthesizer-alpha/test/integ.synth-default-resources.ts

```diff
@@ -4,10 +4,10 @@ import { App, Stack } from 'aws-cdk-lib';
 import * as lambda from 'aws-cdk-lib/aws-lambda';
 import { BucketEncryption } from 'aws-cdk-lib/aws-s3';
 import { APP_ID_MAX } from './util';
-import { AppStagingSynthesizer } from '../lib';
+import { AppStagingSynthesizer, BootstrapRole, DeploymentIdentities } from '../lib';
 
 // IMAGE_COPIES env variable is used to test maximum number of ECR repositories allowed.
-const IMAGE_COPIES = Number(process.env.IMAGE_COPIES) ?? 1;
+const IMAGE_COPIES = Number(process.env.IMAGE_COPIES) || 1;
```

Comment:
If we use `??`, the right side will not be returned if `Number(~)` returns NaN when the IMAGE_COPIES is undefined.